bosh/opsfiles/rules.yml

```diff
@@ -313,3 +313,18 @@
       annotations:
         summary: 'Domain broker {{$labels.instance}} is close to the certificates per listener limit: {{$value}}'
         description: Provision more load balancers for domain broker {{$value}}
+
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: blackbox
+    rules:
+    - alert: SSLCertExpiringSoon
+      expr: probe_ssl_earliest_cert_expiry{job="blackbox"} - time() < 86400 * 30
+      for: 10m
+      labels:
+        service: blackbox
+        severity: warning
+      annotations:
+        summary: SSL certificate for {{$labels.instance}} is expiring within 30d
```

Comment:
Since we're using letsencrypt across the board now, and the absolutely earliest you can renew is 30 days from expiration, shouldn't we check this at like 20 or 25 days to give it a chance to renew first? I have a feeling if we continue to see this alert for something that regularly happens every 90 days it will become nothing but noise.